modules/preview-react/pill/lib/usePillModel.tsx

```diff
@@ -0,0 +1,40 @@
+import {createEventMap, Model, ToModelConfig, useEventMap} from '@workday/canvas-kit-react/common';
+
+type PillState = {
+  maxWidth?: number | string;
+  onClick?: () => void | undefined;
+  onDelete?: () => void | undefined;
+  disabled?: boolean;
```

Comment:
disabled is used for `removable` pills since they're span elements and the `disabled` property doesn't exist on a `span` element. I use this to style them correctly